### PR TITLE
Province membership for tasks

### PR DESCRIPTION
**Pull request by geohacker**
Fixes https://github.com/orma/openroads-vn-analytics/issues/213

### CODE REVIEW

File: scripts/make-analysis-and-display.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ tippecanoe \
 
 export MapboxAccessToken=$MAPBOX_ACCESS_TOKEN
 mapbox-upload \
-    "${MAPBOX_ACCOUNT}.vietnam-conflated" \
+    "${MAPBOX_ACCOUNT}.vietnam-conflated-1" \
```
Comment (kamicut) [inline scripts/make-analysis-and-display.sh:62]: What's this "-1" for?

**Review by kamicut — COMMENTED** [review]
LGTM but needs clarification

File: scripts/make-analysis-and-display.sh (hunk 1 above)
Comment (geohacker) [reply scripts/make-analysis-and-display.sh:62]: @kamicut this PR is based off of master. We lost the .vietnam-conflated map id due to a processing bug which I need mapbox support to release. Until then we're using the -1 map id, the front-end already points to this.

Comment (kamicut) [reply scripts/make-analysis-and-display.sh:62]: 👍

**Pull request merged**